.eslintrc

```diff
@@ -13,6 +13,11 @@
   "parser": "babel-eslint",
   "rules": {
     "arrow-parens": ["error", "always"],
+    // Beware about turning this rule back on. The rule encourages you to create
+    // static class methods on React components when they don't use `this`.
+    // However, it will not warn you if you access `this` in a static method
+    // accidentally (which would most likely be a mistake).
+    "class-methods-use-this": "off",
```

Comment:
yeah, I was playing with this too. I can't think of any case where we'd *want* to make a react component's method static if that method is used by render(). Maybe AirBnB had this rule because they don't use lifecycle react classes? I know stateless components is all the hotness.